api/Animation.json

```diff
@@ -342,7 +342,8 @@
               "version_added": false
             },
             "safari": {
-              "version_added": false
+              "version_added": "10",
+              "version_removed": "11"
```

Comment:
Was Animation.effect really removed?

Reply:
In latest Safari TP (so beyond 11), the Animation interface isn't exposed.